ui/components/failed.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import ImageDiff from 'react-image-diff';
+import {Clearfix, ButtonGroup, Button,
+  Jumbotron, Grid, Row, Col, Panel}
+  from 'react-bootstrap';
+
+function _renderDefaultView(diff) {
+  return <div className="simple">
+      <img className="diff"
+           src={diff}
+           key={diff}
+      />
```

Comment:
You can do `key={baseline} />` with the closing tag on the same line and save some lines. Having it on a separate line doesn't bring that much value.

Same goes for all the places indented like this 🌵